File: Finde Max Value/FindMax.java
class FindMax {
    public static void main(String[] args) {
        int[] list = {15, 10, 12, 33, 56, 22};

        int index = 0; 
        int max = list[index];

        for (int i : list) {
            if (i > max) {
                max = list[index];      
            }

            index++;
        }
        System.out.printf("The max value of our list is %d", max);
    }
}

File: Finde Max Value/FindMax.swift
import UIKit

func findMax(arr:[Int]) -> Int{
    var max: Int = arr[0]
    for i in arr {
        if i > max {
            max = i
        }
    }
    return max
}

let arr1:[Int] = [15, 10, 12, 33, 56, 22]
print(findMax(arr: arr1))
